db/migrate/20240903024211_add_trigger_to_users_to_sync_user_passwords.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+class AddTriggerToUsersToSyncUserPasswords < ActiveRecord::Migration[7.1]
+  def up
+    # necessary for postgres < v14 which does not have CREATE OR REPLACE TRIGGER
```

Comment:
Is this a real concern? I really hope that most of the self hosters have already upgrades to Postgres v15.

Reply:
Most self hosters are still running PG 13 as we have yet to bump the default base image to PG 15.